paddle/fluid/platform/cudnn_helper.h

```diff
@@ -62,7 +62,7 @@ inline const char* cudnnGetErrorString(cudnnStatus_t status) {
 
 #define CUDNN_ENFORCE(condition)                                     \
   do {                                                               \
-    cudnnStatus_t status = condition;                                \
+    auto status = condition;                                         \
```

Comment:
why make this change?

Reply:
NVCC on windows can trigger a type translation error from size_t to cudnnstatus_t, let the compiler to decide the type can fix that.